src/Agent.Plugins/BuildServer.cs

```diff
@@ -42,5 +44,15 @@ public BuildServer(VssConnection connection)
 
             return await _buildHttpClient.CreateArtifactAsync(artifact, projectId, buildId, cancellationToken: cancellationToken);
         }
+
+        // Get named artifact from a build
+        public async Task<BuildArtifact> GetArtifact(
+            Guid projectId,
+            int buildId,
+            string name,
+            CancellationToken cancellationToken = default(CancellationToken))
+        {
+            return await _buildHttpClient.GetArtifactAsync(projectId, buildId, name, null, cancellationToken);
```

Comment:
what is the null for? you might want to use named parameter

Reply:
added